examples/source_stream_wfs_25d.html

```diff
@@ -230,19 +235,26 @@
             var wfsBuildingLayer = new itowns.GeometryLayer('wfsBuilding', new itowns.THREE.Group(), {
                 update: itowns.FeatureProcessing.update,
                 convert: itowns.Feature2Mesh.convert({
-                    color: colorBuildings,
-                    batchId: function (property, featureId) { return featureId; },
-                    altitude: altitudeBuildings,
-                    extrude: extrudeBuildings }),
+                    batchId: function (property, featureId) { return featureId; }
+                }),
                 onMeshCreated: function scaleZ(mesh) {
-                    mesh.scale.z = 0.01;
-                    meshes.push(mesh);
+                    mesh.children.forEach(c => {
+                        c.scale.z = 0.01;
+                        meshes.push(c);
+                    })
```

Comment:
> For my information : is this part combined with the scaler method defined from lines 199 to 212 responsible for the buildings 'rising from the ground' effect ?

yes

Reply:
> Also : what is the difference between applying a scale to the whole meshes group instead of applying it individually to each mesh within the group ?

because the mesh `Feature` is added to new `Group`.

but yes you're right, it's not very intuitive
I will look for a way to avoid this

Reply:
Ok, thanks !

Reply:
We keep it for the moment.